library/auth.inc

```diff
@@ -46,6 +46,7 @@ if (isset($_GET['auth']) && ($_GET['auth'] == "login") && isset($_POST['authUser
     else {
         $_SESSION['language_choice'] = 1;
     }
+    $_SESSION['language_direction'] = getLanguageDir($_SESSION['language_choice']  );
```

Comment:
Note that auth.inc script is not always called. Would it make sense to instead set $GLOBALS['css_header'] in the interface/globals.php script that is always called:
https://github.com/matrix-olegsv/openemr/blob/11082dc57af94ccc750936dcb983a9b851721d02/interface/globals.php#L215-L217